### DISCUSSION TITLE
Input data storage

### DISCUSSION THREAD

**Discussion by evalieungh** [NorESMhub/noresm-land-sites-platform#143] [General]
"The only constant is change."

Our input data is stored on NIRD (<https://ns2806k.webs.sigma2.no/EMERALD/EMERALD_platform/inputdata_fates_platform/>). 

@lasseke just got notified our project period is running out and needs renewal to keep the files up. Should we keep the files there, or are there more stable alternatives? The files are not overly large, so maybe Zenodo or Open Science Framework repositories could be alternatives. Or somewhere else.

I think we should keep the files somewhere accessible that requires the least amount of maintenance, and least hassle with uploading new versions. And it needs to be open and accessible for the foreseeable future, of course.

**Comment by huitang-earth** [comment]
I would vote for keeping it in OSF so that it is in line with the observation data from Bergen group......But, I don't know what is the strategy for the observation data in LATICE sites or other sites.

**Comment by evalieungh** [comment]
OSF seems like a safe enough option. From their FAQ page: "_How can it be free? How are you funded?_
OSF is maintained and developed by the [Center for Open Science](http://cos.io/) (COS), a 501(c)3 non-profit organization. COS is supported through grants from a variety of supporters, including [federal agencies, private foundations, and commercial entities](https://www.cos.io/about/supporters). Read more about [our finances](https://cos.io/about/our-finances/).
_What if you run out of funding? What happens to my data?_
COS established a $250,000 preservation fund for hosted data in the event that COS had to curtail or close its offices. If activated, the preservation fund will preserve and maintain read access to hosted data. This fund is sufficient for 50+ years of read access hosting at present costs. COS will incorporate growth of the preservation fund as part of its funding model as data storage scales. For information about OSF backups and technical preservation details, see the [OSF Backup and Preservation Policy](https://help.osf.io/article/203-faqs#Backup-Preservation-Policy)."

The next question is how easy it is to upload new data versions there after creating it on e.g. Saga. Data upload is easy on OSF in general, but I'm not sure how automated we can make it compared to NIRD.

**Comment by evalieungh** [comment]
We just discussed this a bit more, and another good option might be to make a new GitHub repository that is linked to zenodo. That way all the LSP stuff will be stored and get DOI's in the same way, and it might be easier for e.g. new users that want to push new sites or input data to the LSP to use GitHub.

**Comment by evalieungh** [comment]
Now we have data in <https://github.com/NorESMhub/noresm-lsp-data> which will get a DOI on Zenodo once we make a release.

**Reply by evalieungh** [reply]
Update: we now have a release and [files on Zenodo](https://zenodo.org/record/7308232).
